modules/ROOT/pages/am-installing.adoc

```diff
@@ -92,6 +93,9 @@ Ensure that one of the following Mule Runtime Manager agent versions is installe
 
 Mule deployments must be visible and managed in the Runtime Manager user interface. To ensure they appear, use the Runtime Manager user interface or the ARM REST API for new deployments.
 
+[IMPORTANT]
+Runtime Manager agent 2.7.x is compatible with all HSA flavors, including GovCloud and PCE. However, FIPS isn't certified. If you need to enable FIPS, skip this version.
```

Comment:
Please clarify and follow guidelines on these:

* HSA - what does that refer to; is it an official way that we refer to that?
* GovCloud - see style guidelines
* PCE: - get clarification about conflicting guidelines in the term ref

Reply:
Did you try using writer.ai. I asked Hanna about PCE and updated the term ref.